Source/control.cpp

```diff
@@ -1273,29 +1226,26 @@ static void CPrintString(CelOutputBuffer out, int y, const char *str, bool cente
 	}
 }
 
-static void PrintInfo(CelOutputBuffer out)
+static void PrintInfo(const CelOutputBuffer &out)
 {
-	int yo, lo, i;
+	if (talkflag)

```

Comment:
Change to early return also unrelated?

Reply:
Suppose, but it made it much easier to reason about the scope of the variables, clang-tidy doesn't like lines of uninitialized variables so had to distribute them.